core/src/edu/asu/c3simulator/screens/RetiredProducts.java

```diff
@@ -0,0 +1,206 @@
+package edu.asu.c3simulator.screens;
```

Comment:
This class has a lot of mistakes similar or identical to other cases. I am postponing the review of this class until it is refactored to address the relative issues mentioned elsewhere in this review.